### DISCUSSION TITLE
Don't use FFD points

### DISCUSSION THREAD

**Discussion by eduVital** [mdolab/dafoam#341] [General]
Hi,

I want to do a simulation in "runAdjoint" mode in order to get a sensitivity map. However, I would like it to be in every node/cell of a patch, instead of in FFD points, in a way that I defined a "Field" design variable. None of the tutorial examples don't use FFD however, is it even possible to do that?

Thanks for the help!

**Comment by friedenhe** [comment]
Check https://github.com/mdolab/dafoam/blob/fc34405582120f5f57173e2af2de7aa8c301520e/tests/runTests_DASimpleFoamAD.py#L47
You don't need to define a field variable, but you do need to define an FFD (just as a template). It will compute the sens map for surface nodes, not FFD.
